src/Servers/Kestrel/Core/ref/Microsoft.AspNetCore.Server.Kestrel.Core.netcoreapp3.0.cs

```diff
@@ -723,7 +727,7 @@ public abstract partial class HttpProtocol : Microsoft.AspNetCore.Http.Features.
     }
     public sealed partial class HttpRequestHeaders : Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpHeaders
     {
-        public HttpRequestHeaders() { }
+        public HttpRequestHeaders(Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions serverOptions) { }
```

Comment:
Just pass a bool? I'm not sure we should pass all the options (though I know where could be more in the future...)

Reply:
Changed